### ISSUE TITLE
[ExpansionPanel] Invalid prop `overflown` of type `number` supplied to `ExpansionPanel`, expected `boolean`

### ISSUE THREAD

**Issue by ostrebler** [mlaursen/react-md#645]
### Description
When nesting ExpansionLists, I get this warning on the inner ExpansionPanels : 
```
warning.js:33 Warning: Failed prop type: Invalid prop `overflown` of type `number` supplied to `ExpansionPanel`, expected `boolean`.
    in ExpansionPanel (created by PatentSelector)
    in ExpansionList (created by PatentSelector)
    in div (created by PanelContent)
    in div (created by PanelContent)
    in PanelContent (created by Motion)
    in Motion (created by Collapse)
    in Collapse (created by ExpansionPanel)
    in li (created by Paper)
    in Paper (created by ExpansionPanel)
    in ExpansionPanel (created by PatentSelector)
    in ul (created by ExpansionList)
    in ExpansionList (created by PatentSelector)
```

The code is a bit long and might not be usefull in that case. As the doc advises, I never set overflown by myself, so this seems to be an intern error. Basically, I just have a structure like this : 
```
<ExpansionList>
  <ExpansionPanel ...>
    <ExpansionList>
    { patents.map(patent =>
         <ExpansionPanel key={ patent.key } ...>
         ...
         </ExpansionPanel>
    )}
    </ExpansionList>
  </ExpansionPanel>
</ExpansionList>
```

### Version
- React - latest
- React-MD - latest

**Comment by mlaursen** [comment]
Interesting. Might be something I close into the children... I'll check into it

**Comment by ostrebler** [comment]
I solved it by changing isOverflown in ExpansionList. It returned 0, I just changed it to 'false'.

**Comment by mlaursen** [comment]
Oh, thanks! I'm actually wondering why I did `0` or a boolean for those...

**Comment by mlaursen** [comment]
I've also realized that my logic doesn't really account for nested `ExpansionPanel`s. Does it still display as expected?

**Comment by ostrebler** [comment]
No, it also didn't display as expected. The panel header used to overflow its box on first expand. Like this : 
![bug-panel](https://user-images.githubusercontent.com/30187797/33848523-4ed53580-deae-11e7-9533-2bca154d7661.png)

I added a `headerClassName` to the panel to set `min-width: 0 !important`

**Comment by mlaursen** [comment]
Going to close this one now since the main PropType warning is fixed. The remaining fixes will go into #642
